config/services.yaml

```diff
@@ -1634,3 +1634,11 @@ services:
     arguments:
       - '@AppBundle\LoopEat\Client'
       - '@monolog.logger.loopeat'
+
+  AppBundle\Transporter\Proximus\ProximusImporter:
+    lazy: true
+    arguments:
+      - "%env(PROXIMUS_SYNC_URI)%"
```

Comment:
We may want to add `PROXIMUS_SYNC_URI` to `.env.dist`?
With an empty value, but just to have it defined there so we all know that it exists!